src/sentry/api/endpoints/monitor_checkins.py

```diff
@@ -68,7 +69,13 @@ def get(
             if project.organization.slug != organization_slug:
                 return self.respond_invalid()
 
-        queryset = MonitorCheckIn.objects.filter(monitor_id=monitor.id)
+        start, end = get_date_range_from_stats_period(request.GET)
```

Comment:
change this to `get_date_range_from_params()`. that's actually a function I added months ago that never made the full transition in docs

Reply:
ahhh very nice will change